mvp-spec/ad-server-implementation.md

```diff
@@ -0,0 +1,772 @@
+# Ad Server Implementation
+
+## Goal of the document
+
+This document provides the guidance and the requirements for implementing 
+Prebid Addressability Framework (PAF) on an Ad Server.
+
+## Overview
+
+For performance purposes, it is recommended to implement the 
+Addressable Content Service directly in the Ad Server. Therefore, the following
+document is focused on this setup.
+
+Compared to a usual setup with an Ad Server, the Publisher or the Provider of
+the Ad Server has to implement additional features to enable PAF. 
+
+Those features are:
+* The generation of the Seeds of the Addressable Contents;
+* The emission of Transmissions to the Ad Network;
+* The support of the Audit Log for the user;
+* The exposition of an Identity endpoint.
+
+The following diagram introduces an overview of this setup:
+
+<!--partial-begin { "files": [ "ad-server-flow.mmd" ], "block": "mermaid" } -->
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+```mermaid
+sequenceDiagram
+    participant User
+    participant Publisher
+    participant AdServer as Ad Server
+    participant DSP
+
+    User->>Publisher: Ask for a webpage
+    Publisher->>AdServer: Offer Inventory<br />with Prebid SSO Data
+    AdServer->>AdServer: Generate Seeds for <br /> the Addressable Contents
+    AdServer->>DSP: Generate and send a <br /> Transmission Request
+    DSP->>AdServer: Provide an Addressable Contents <br /> and a Transmission Response
+    AdServer->>Publisher: Provide the Addressable Contents
+    Publisher->>User: Display the Addressable Contents <br /> and their respective Audit Logs
+
+```
+<!--partial-end-->
+
+## Offer inventory with Prebid SSO Data
+
+PAF doesn't standardize the API of the Ad Server for offering
+inventory because each Ad Server has an existing and specific API. 
+However, to implement PAF, the Ad Server need to implement new features
+in the existing endpoints called by the Publisher website to offer inventory 
+and get back Addressable Contents.
+
+To understand the steps, it is important to overview how to generate the data and
+the relationships between them:
+* A Publisher offers *multiple* placements - one for each for 
+Addressable Contents - via an Ad Server.
+* An Ad Server generates *one* Seed for each Addressable Content
+* An Ad Server generates and sends *one* Transaction Request per Seed and Supplier
```

Comment:
Here shouldn't be transmission instead of transaction as laid out in the diagram below?